sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -195,7 +195,9 @@ export class EventHubConsumerClient {
    * <yournamespace>.servicebus.windows.net
    * @param eventHubName - The name of the specific Event Hub to connect the client to.
    * @param credential - An credential object used by the client to get the token to authenticate the connection
-   * with the Azure Event Hubs service. See &commat;azure/identity for creating the credentials.
+   * with the Azure Event Hubs service.
+   * See &commat;azure/identity for creating credentials that support AAD auth.
+   * See &commat;azure/core-auth for creating `AzureNamedKeyCredential` or `AzureSASCredential`.
```

Comment:
This worked for `@azure/identity` because its readme had all the info about the different credentials. Not so much for core-auth. Can we update the readme for core-auth to include code snippets on how to create and update these credentials?

Reply:
Sure.

Reply:
So, after some more thought I instead improved the documentation in event hubs. The reason being, for _event-hubs_ you'd pass the SharedAccessKeyName and SharedAccessKey to AzureNamedKeyCredential, but this may be different for other services in the future.

Rather than try to make a generic example in core-auth, I tried to be more explicit about what the user would need to pass in.